summaries/sprint2.md

```diff
@@ -0,0 +1,24 @@
+# Week 1 sprint retrospective
+
+## Scrum master (Konrad, Jonas)
+This week we vastly improved on time management in meetings. They were much shorter and a lot more efficient.
+We had two short standups (Monday and Thursday) during which everyone updated the group on their current status and their dependencies on work from other people. After the meeting these dependencies where resolved by the concerned parties.
+Average meeting duration was 15 minutes.
+Pull request were there faster but not entirely ready for merging on Wednesday because of troubles writing the tests.
+
+## Bettens, Louis

```

Comment:
```suggestion
## Bettens, Louis
This week I provided a well-modularised implementation of the cloud database back-end of the app. This required discussing the interface of the module with the rest of the team, in particular with Martin who provided it. I also had to fight the CI to get acceptable test coverage on my work due to odd issues with the Glide library, which took a lot of time at the end of the process and almost caused a failure to deliver on time.

I noted that, since Martin and I miscommunicated, it might be preferable in the future to structure such discussions around trying to write down a decision in an electronic document, so as to make sure everything is clear and can be referred back to.
```

Here's my retrospective